### ISSUE TITLE
Scraper does not download content when using "auto_model_choice": "all"

### ISSUE THREAD

**Issue by TheMissingPort** [DIGITALCRIMINALS/OnlyFans#735]
The scraper runs but its not actually scraping any content.

It will randomly download one or two pictures but thats it.

Any suggestions?

Thanks

**Comment by TheMissingPort** [comment]
EDIT: When I changed it from scraping all models automatically, it seems to work. So I can't autoscrape all models

**Comment by meanjoep92** [comment]
> EDIT: When I changed it from scraping all models automatically, it seems to work. So I can't autoscrape all models

Currently facing this exact same issue. How exactly did you accomplish this? (Did you have to change any settings?)

**Comment by TheMissingPort** [comment]
> > EDIT: When I changed it from scraping all models automatically, it seems to work. So I can't autoscrape all models
> 
> Currently facing this exact same issue. How exactly did you accomplish this? (Did you have to change any settings?)

auto_media_choice = “” 

I had it to “ALL”. When I changed it back to blank it seems to work now

**Comment by meanjoep92** [comment]
> > > EDIT: When I changed it from scraping all models automatically, it seems to work. So I can't autoscrape all models
> > 
> > 
> > Currently facing this exact same issue. How exactly did you accomplish this? (Did you have to change any settings?)
> 
> auto_media_choice = “”
> 
> I had it to “ALL”. When I changed it back to blank it seems to work now

Thank you! Alas, it still didn't work on my end. At this point, I'm lost on what to do.

**Comment by TheMissingPort** [comment]
> > > > EDIT: When I changed it from scraping all models automatically, it seems to work. So I can't autoscrape all models
> > > 
> > > 
> > > Currently facing this exact same issue. How exactly did you accomplish this? (Did you have to change any settings?)
> > 
> > 
> > auto_media_choice = “”
> > I had it to “ALL”. When I changed it back to blank it seems to work now
> 
> Thank you! Alas, it still didn't work on my end. At this point, I'm lost on what to do.
> 
> EDIT: The ONLY way I was able to get it working again was to disable my VPN. (I remember this was also a problem months ago) Of course, not trying to scrape without a VPN. Hopefully there's a patch for that.

I can't seem to get it to scrap paid stuff sadly

**Comment by inadquatecoding** [comment]
I'm having this problem too, I will select the number I want and it's ignored and executes the entire loop instead, not downloading anything despite saying it is.

(names changed)

Auth (V1) Attempt 1/10
Welcome harry  | harry

```
Choose Subscriptions: 0 = All | 1 = karine | 2 = thebritt | 3 = caroline | 4 = mark | 5 = tay | 6 = u27355 | 7 = xkatie | 8 = staywild | 9 = court | 10 = maddie | 11 = bethanie | 12 = kinky | 13 = cute | 14 = kate | 15 = mari_ | 16 = mari
Scraping Subscriptions
Scraping Paid Content
14
Scraping - karine | 1 / 16
Scraping - thebritt | 2 / 16
Scraping - caroline | 3 / 16
Scraping - mark | 4 / 16
Scraping - tay | 5 / 16
Scraping - u27355 | 6 / 16
Scraping - xkatie | 7 / 16
Scraping - stay | 8 / 16
Scraping - court | 9 / 16
Scraping - maddie | 10 / 16
Scraping - bethanie | 11 / 16
Scraping - kinky | 12 / 16
Scraping - cute | 13 / 16
Scraping - kate | 14 / 16
Scraping - mari_ | 15 / 16
Scraping - mari | 16 / 16
Scraping Message Content
Scrape Processing
Name: u27355
Type: Messages
Processing Scraped Messages
100%|███████████████████████████████████████████████████████████████████████████████████████████| 16/16 [00:00<00:00, 3214.33it/s]
Processing metadata.
Finished processing metadata.
Renaming files.
100%|█████████████████████████████████████████████████████████████████████████████████████████████| 8/8 [00:00<00:00, 3550.36it/s]
Scrape Completed

Scrape Processing
Name: yhes
Type: Messages
Processing Scraped Messages
100%|███████████████████████████████████████████████████████████████████████████████████████████| 17/17 [00:00<00:00, 1310.89it/s]
Processing metadata.
Finished processing metadata.
Renaming files.
100%|███████████████████████████████████████████████████████████████████████████████████████████| 12/12 [00:00<00:00, 6509.53it/s]
Scrape Completed

```

Executed 14 to download katie, but it starts just processing everything and it says it's downloading but doesn't seem to be

**Comment by kccomikeyga** [comment]
Can confirm same issue on most recent master build with clean install of Windows, Python 3.10.9 and all prerequisites.

Trying to dig into it.

**Comment by Bearded-Beard** [comment]
Mine worked until I did a update script now it'll hang on a profile for hours, I wish I still had my previous version

**Comment by kccomikeyga** [comment]
Try this:  https://github.com/DIGITALCRIMINALS/OnlyFans/issues/679#issuecomment-1356768986

**Comment by DIGITALCRIMINALS** [comment]
Try this branch

**Comment by MillerTimeJones** [comment]
I have this issue even without vpn running and tried the latest test version.

**Comment by dpuT5** [comment]
Yea I regret updating now. I had the old version before Ultima update and it scraped everything fine. Not sure why @DIGITALCRIMINALS changed everything

**Comment by DIGITALCRIMINALS** [comment]
> Yea I regret updating now. I had the old version before Ultima update and it scraped everything fine. Not sure why @DIGITALCRIMINALS changed everything

Ironically, I'm updating the script to make it easier to fix bugs and implement new features. The script used to be monolithic, but I made it more modular and streamlined and it has helped a lot.

Anyway, my needs for the script has evolved, so I had to update it.
I wouldn't use this script until I release v8. It is what it is.

I would suggest using something simpler, like:

- https://github.com/taux1c/onlyfans-scraper
- https://github.com/k0rnh0li0/onlyfans-dl
